gdnative-derive/src/native_script.rs

```diff
@@ -36,12 +36,20 @@ pub(crate) fn derive_native_class(input: TokenStream) -> TokenStream {
                 None
             };
 
+            let update_config_warning_on_set_snippet = match config.update_config_warning_on_set {
+                true => quote!(_owner.update_configuration_warning();),
+                false => quote!(),
+            };
```

Comment:
Shouldn't this just be an `if` expression...?

Reply:
I like the way match forces handling all possibilities and allows just setting the variable appropriately.

Reply:
`if` is an expression.

```rust
let update_config_warning_on_set_snippet = if config.update_config_warning_on_set {
    quote!(_owner.update_configuration_warning();)
} else {
    quote!()
};
```